x-pack/plugins/observability_solution/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -141,7 +141,7 @@ ELASTIC_AGENT_STATE="$(elastic-agent status --output json | sed -n '/components/
 # Get elastic-agent status in json format | removing extra states in the json | finding "message":value | removing , | removing "message": | trimming the result | removing ""
 ELASTIC_AGENT_MESSAGE="$(elastic-agent status --output json | sed -n '/components/q;p' | grep message | sed 's/\(.*\),/\1 /' | sed 's/"message": //' | sed 's/\s//g' | sed 's/\"//g')"
 # Get elastic-agent status in json format | removing extra ids in the json | finding "id":value | removing , | removing "id": | trimming the result | removing ""
-ELASTIC_AGENT_ID="$(elastic-agent status --output json | sed -n '/components/q;p' | grep id | sed 's/\(.*\),/\1 /' | sed 's/"id": //' | sed 's/\s//g' | sed 's/\"//g')"
+ELASTIC_AGENT_ID="$(elastic-agent status --output json | sed -n '/components/q;p' | grep \"id\" | sed 's/\(.*\),/\1 /' | sed 's/"id": //' | sed 's/\s//g' | sed 's/\"//g')"
```

Comment:
While you're here, could you please also change the `sed 's/\s//g'` in all three to `sed 's/[[:space:]]//g'`? Right now trimming does not work and breaks the `if` on line `145`.

Reply:
TIL about `[[:space:]]`, interesting